fudge/Cargo.toml

```diff
@@ -15,4 +15,5 @@ sc-executor = { workspace = true }
 sp-io = { workspace = true }
 
 [features]
+default = []
```

Comment:
I would add to the above dependencies the `default-features = false`, if not, we're using the non-std in the crates that use this crate as a dependency

Reply:
@cdamian

Reply:
Done.